_posts/2017-08-30-Modernizing-Force.markdown

```diff
@@ -0,0 +1,306 @@
+---
+layout: post
```

Comment:
Would recommend making that "layout: epic" - I might make a danger rule for this